Index: ffmpeg-5.0/libavformat/tls_gnutls.c
===================================================================
--- ffmpeg-5.0.orig/libavformat/tls_gnutls.c	2022-01-14 19:45:40.000000000 +0100
+++ ffmpeg-5.0/libavformat/tls_gnutls.c	2022-02-10 23:06:52.364362893 +0100
@@ -192,7 +192,7 @@ static int tls_open(URLContext *h, const
     gnutls_transport_set_pull_function(p->session, gnutls_url_pull);
     gnutls_transport_set_push_function(p->session, gnutls_url_push);
     gnutls_transport_set_ptr(p->session, p);
-    gnutls_priority_set_direct(p->session, "NORMAL", NULL);
+    gnutls_set_default_priority(p->session);
     do {
         if (ff_check_interrupt(&h->interrupt_callback)) {
             ret = AVERROR_EXIT;
